src/odb/src/db/dbPowerSwitch.cpp

```diff
@@ -240,16 +237,142 @@ void dbPowerSwitch::addOnState(const std::string& on_state)
   obj->_on_state.push_back(on_state);
 }
 
+void dbPowerSwitch::addLibCell(const std::string& lib_cell)
```

Comment:
Can a power switch have more than one cell?

Reply:
According to my understanding of the spec not really. Are you referring to the word "add" instead of "set"?

Reply:
yes